lib/lita/handlers/jira_issues.rb

```diff
@@ -10,6 +10,8 @@ class JiraIssues < Handler
       config :password, required: true, type: String
       config :ignore, default: [], type: Array
       config :issue_ttl, default: 0, type: Integer
+      config :format, default: "[%I] %t\nStatus: %s, assigned to %a, rep. by %r, fixVersion: %v, priority: %P\n%U", type: String
```

Comment:
Yes, I finally saw the results from Travis CI. Yes, you are correct that %p would fix most of the failures. 

I have actually been thinking about how to change the substitutions for how to do uppercase, lowercase, proper case and the original value. So I will probably be changing the substitutions soon.